src/pibasis.jl

```diff
@@ -31,8 +31,8 @@ end
 
 
 function PIBasisSpec( basis1p::OneParticleBasis,
-                      symgrp::SymmetryGroup, 
```

Comment:
we may need to figure out now how to avoid your editor removing white space since this really pollutes the git history.

Reply:
in particular this makes it more difficult for me to review the changes.

Reply:
Indeed that is a problem. Sorry about that... It is more than worse when I found that I can't even add the space back manually either. I've searched and checked the ATOM setting issue but still failed to fix it. Maybe I will need to migrate to vs code soon.

Reply:
Thanks for fixing the white space issues